index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="main.css">
+  <title>Apple</title>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
+</head>
+<body>
+  <header class="main-header">
+    <nav class="main-nav-bar">
+      <ul class="menu-bar">
+        <li class="logo menu"><a href="#"><ion-icon name="logo-apple" class="main-logo"></ion-icon></a></li>
+        <li class="menu"><a href="#">Store</a></li>
+        <li class="menu"><a href="#">Mac</a></li>
+        <li class="menu"><a href="#">iPod</a></li>
+        <li class="menu"><a href="#">iPhone</a></li>
+        <li class="menu"><a href="#">iPad</a></li>
+        <li class="menu"><a href="#">iTunes</a></li>
+        <li class="menu"><a href="#">Support</a></li>
+        <li class=" search"><input type="text" class="fontAwesome search-area" placeholder="&#xF002;" value=""></li>
+      </ul>
+    </nav>
+    <div class="header-contents">
+      <p class="verse">
+        <a href="#">What will your verse be?</a>
+      </p>
+      <p class="something-to-share">
+        <a href="#">
+          Everyone has something to share. See how people are using<br>
+          iPad in their unique ways to add to the world's story.
+        </a>
+      </p>
+      <p class="your-verse">
+        <a href="#">Watch the "Your Verse" film and explore the stories</a>
+        <img src="images/your-verse-Copy.png" alt="play">
+      </p>
+    </div>
+  </header>
+  <main class="main-content">
+      <div class="promo-img">
+        <img src="images/featured-ad1.jpg" alt="iphone 5s">
+      </div>
+      <div class="promo-img">
+        <img src="images/featured-ad2.jpg" alt="iPhone 5c">
+      </div>
+      <div class="promo-img">
+        <img src="images/featured-ad3.jpg" alt="Happy bday, mac">
+      </div>
+      <div class="promo-img">
+        <img src="images/featured-ad4.jpg" alt="supplier responsibility">
+      </div>
+  </main>
```

Comment:
- [ ] these should be links (to other articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))